rpc/documentation/api.md

```diff
@@ -466,47 +467,95 @@ or upon received an error.
 - `PeerNotification peer_information`: This contains information about the
   current state of the wallet's peers.
 
-- `SyncingStatus syncing_status`: Various properties to describe the current
-  state of syncing the wallet is currently performing.  Once synced, this will
-  be set to nil.
+**Expected Errors:**
+
+- `FailedPrecondition`: The wallet or consensus RPC server has not been opened.
+  The private passphrase does not successfully unlock the wallet.  The string
+  provided to spv_connect is not a valid address or port.
+
+- `InvalidArgument`: The private passphrase is incorrect.
```

Comment:
Ambiguous whether a wrong passphrase will return `InvalidArgument` or `FailedPrecondition` (given that the previous error is also returned in case of "private passphrase does not successfully unlock the wallet")